src/coreclr/jit/codegencommon.cpp

```diff
@@ -8588,6 +9373,178 @@ void CodeGen::genFnEpilog(BasicBlock* block)
     }
 }
 
+#elif defined(TARGET_LOONGARCH64)
+
+void CodeGen::genFnEpilog(BasicBlock* block)
```

Comment:
Move this function to codegenloongarch64.cpp (I am going to move the others to arch-specific files)

Reply:
OK, Thanks